deploy/crds/planetscale.com_vitessclusters.yaml

```diff
@@ -1992,6 +1992,10 @@ spec:
                                             type: object
                                           mysqldExporter:
                                             properties:
+                                              extraFlags:
+                                                additionalProperties:
+                                                  type: string
+                                                type: object
```

Comment:
The operator.yaml file has to be re-generated to take into account the updated CRDs. It needs to be updated in two places, in the tests, and in the tests of the vitess repository.

### vtop tests
You can use `make generate && kustomize build ./deploy > build/_output/operator.yaml` to generate the new yaml file and copy it over to `test/endtoend/operator/operator-latest.yaml`. I tried piping it directly to the test file, but it reformats a lot of lines that we do not want to reformat.

Once copied, there are a couple line changes you will want to revert, starting from line `7702`, the end of the file should look like the snippet below:
```yaml
          image: vitess-operator-pr:latest
          imagePullPolicy: Never
          name: vitess-operator
          resources:
            limits:
              memory: 512Mi
            requests:
              cpu: 100m
              memory: 128Mi
      priorityClassName: vitess-operator-control-plane
      serviceAccountName: vitess-operator
```

### vitess repository
Once the `test/endtoend/operator/operator-latest.yaml` file is updated, you will need to copy it all over to `./examples/operator/operator.yaml` in the vitess repository, [here](https://github.com/vitessio/vitess/blob/main/examples/operator/operator.yaml). And open a PR. Both PRs, the vitess one and this one, should be merged around the same time.

On the vitess PR, the end of the file shouldn't change, and still be as follows:
```yaml
          image: planetscale/vitess-operator:latest
          name: vitess-operator
          resources:
            limits:
              memory: 512Mi
            requests:
              cpu: 100m
              memory: 128Mi
      priorityClassName: vitess-operator-control-plane
      serviceAccountName: vitess-operator
```